hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieBaseFileGroupRecordBuffer.java

```diff
@@ -96,6 +98,12 @@ public HoodieBaseFileGroupRecordBuffer(HoodieReaderContext<T> readerContext,
     this.payloadProps = payloadProps;
     this.internalSchema = readerContext.getSchemaHandler().getInternalSchema();
     this.hoodieTableMetaClient = hoodieTableMetaClient;
+    long maxMemorySizeInBytes = payloadProps.getLong(MAX_MEMORY_FOR_MERGE.key(), MAX_MEMORY_FOR_MERGE.defaultValue());
+    String spillableMapBasePath = payloadProps.getString(SPILLABLE_MAP_BASE_PATH.key(), FileIOUtils.getDefaultSpillableMapBasePath());
```

Comment:
Probably we should not call this `payloadProps` anymore.

Reply:
ok